components/splitter/SplitBar.tsx

```diff
@@ -42,12 +44,16 @@ const SplitBar: React.FC<SplitBarProps> = (props) => {
     onOffsetUpdate,
     onOffsetEnd,
     onCollapse,
+    lazy,
+    containerSize,
   } = props;
 
   const splitBarPrefixCls = `${prefixCls}-bar`;
 
   // ======================== Resize ========================
   const [startPos, setStartPos] = useState<[x: number, y: number] | null>(null);
+  const [previewStyle, setPreviewStyle] = useState<React.CSSProperties>();
```

Comment:
```suggestion
  const [constrainedOffset, setConstrainedOffset] = useState<React.CSSProperties>(null);
```

用这个作为状态，previewStyle 可以通过这个计算出来，constrainedOffset 为 null 时戴代表隐藏。